Composer/packages/tools/language-servers/intellisense/src/utils/intellisenseServerUtils.ts

```diff
@@ -0,0 +1,33 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+import { Position, Range, TextDocument } from 'vscode-languageserver';
+
+export const getRangeAtPosition = (document: TextDocument, position: Position): Range | undefined => {
+  const text = document.getText();
+  const line = position.line;
+  const pos = position.character;
+  const lineText = text.split('\n')[line];
```

Comment:
Consider a check that line is not outside the range of the split result.

Reply:
ok!